src/test/java/api/handlers/CloseDeclaredLostLoanWhenLostItemFeesAreClosedApiTests.java

```diff
@@ -36,6 +38,12 @@ public void createLoanAndDeclareItemLost() {
       .forLoanId(loan.getId()));
   }
 
+  @After
+  public void cleanUp() {
+    notesClient.deleteAll();
+    noteTypeClient.deleteAll();
+  }
```

Comment:
It should not be required now, we have simplified the clean-up logic, and everything is removed by a single destroy tenant call now.

Let me know, if it is not working for you.